packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -273,6 +275,24 @@ export class ZWaveController extends EventEmitter {
 		return this._nodes;
 	}
 
+	/** Returns a reference to the (virtual) broadcast node, which allows sending commands to all nodes */
+	public getBroadcastNode(): VirtualNode {
+		return new VirtualNode(
+			NODE_ID_BROADCAST,
+			this.driver,
+			this.nodes.values(),
+		);
+	}
+
+	/** Creates a virtual node that can be used to send multicast commands to several nodes */
+	public getMulticastGroup(...nodeIDs: number[]): VirtualNode {
```

Comment:
@AlCalzone why did you use the spread operator? I think that pass an array is good here :)

Reply:
I thought being able to do this
```
driver.controller.getMulticastGroup(2, 3, 4)
```
would be nice. Though the case of passing an array is probably more frequent.

Reply:
>  Though the case of passing an array is probably more frequent.